### DISCUSSION TITLE
Neo-tree expand/collapse icon

### DISCUSSION THREAD

**Discussion by RazaGR** [LazyVim/LazyVim#375] [Q&A] [answered]
any idea why I can't see collapse/expanded arrow icon like in below image?

I tried with below but no success

```
return {
  {
    "nvim-neo-tree/neo-tree.nvim",
    opts = {
      indent = {
            with_expanders = true, 
            expander_collapsed = "",
            expander_expanded = "",
            expander_highlight = "NeoTreeExpander",
          }
    },
  }.
}
```
<img width="428" alt="Screenshot 2023-03-03 at 6 43 28 PM" src="https://user-images.githubusercontent.com/30994076/222790948-01d5ad9b-204e-457b-8f73-f10f506a3145.png">

this is what I have
![Screenshot 2023-03-03 at 6 46 06 PM](https://user-images.githubusercontent.com/30994076/222791296-7b5a7b16-569c-45f5-acc1-8db671d75679.png)

**Comment by folke** [comment, marked as answer]
Your config is wrong. See:
![image](https://user-images.githubusercontent.com/292349/222806449-eba46d70-1032-4300-8d26-9f1bece51986.png)

But I also just added that to LazyVim by default, since I like it 😄 (still need to make a new release)

**Reply by RazaGR** [reply]
ah my bad, Thanks for the info and cheers for the great work. Keep it up.

**Comment by RazaGR** [comment]
btw here are a nice key combo to collapse/expand folders if needs to be added t default as well

```
return {
  {
    "nvim-neo-tree/neo-tree.nvim",
    opts = {
      window = {
        mappings = {
          ["<space>"] = "none",
          ["h"] = {
            "close_node",
          },
          ["l"] = {
            "open",
          },
        },
      },
    },
  },
}
```
